apps/shared/account/account.service.ts

```diff
@@ -48,7 +62,9 @@ export class AccountService {
 
   /** Sign the user out of the application. */
   async signOut() {
-    await this.auth.signOut();
+    await this.auth.signOut().then(() => {
+      return this.router.navigate(['login']);
```

Comment:
I don't fully recall the layout- but do we know the route is _always_ called `login`?

Reply:
We do at this point, I put a todo in to make this better.